lms/djangoapps/grades/migrations/0001_initial.py

```diff
@@ -2,19 +2,42 @@
 from __future__ import unicode_literals
```

Comment:
Looks like 0002 is present in release already as well (https://github.com/edx/edx-platform/tree/release/lms/djangoapps/grades/migrations), so none of this migration squashing can happen.